angelz510/react-challenge-project-sep-trial
# fix(order_by): users info added to order

## Changes
1. Changed email value in auth reducer. It had 'login' instead of email.

## Purpose
The problem was that the previous code was not displaying the users name when placing an order in /view-orders.

## Approach
This change now displays users who place order.

## Pre-Testing TODOs
NA

## Testing Steps
When placing an order go to /view-orders and check 'ordered by' and it will display log in name. 

Bug: Retrieve user from Redux and include in Order Form
Closes #004
